### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by badsyntax** [galenmaly/lighterpack#118]
Hi, I'd like to know the status of the project, and if the maintainer is still accepting contributions?

I'd like to contribute but I'm not sure if I'm wasting my time if the maintainer is not accepting contributions.

I'm willing to help maintain the project and review and accept contributions if that would help.

Thanks for the useful tool btw!

**Comment by stefandesu** [comment]
Hi, a different developer decided to take this on recently, you can read their post here: https://www.reddit.com/r/Ultralight/comments/be9ba2/lighterpack_i_am_a_developer_and_will_begin_to/

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by badsyntax** [comment]
Thanks! I have added a comment to the thread. I'll keep this issue open for a bit as it's good for visibility

**Comment by galenmaly** [comment]
Thanks for your interest in contributing! I've added a contributing guide that elaborates the project status: https://github.com/galenmaly/lighterpack/blob/master/CONTRIBUTING.md

TL;DR is that we need help with getting the rewrite out the door before any new feature work can begin, and would love any and all help with that.

**Comment by romelwell** [comment]
Galen,. Thanks for putting this out!
